src/nvim/os/env.c

```diff
@@ -557,6 +557,27 @@ char *vim_getenv(const char *name)
     if (p_hf != NULL && vim_strchr(p_hf, '$') == NULL) {
       vim_path = (char *)p_hf;
     }
+
+#ifdef WIN32
+    // Find runtime path relative to the nvim binary i.e. ../share/runtime
+    char exe_name[MAXPATHL];
+    char abs_runtime_path[MAXPATHL];
+    if (vim_path == NULL) {
+      if (os_exepath(exe_name, MAXPATHL) == 0) {
+        // Remove trailing bin/nvim.exe
+        char *path_end = (char *)path_tail_with_sep((char_u *)exe_name);
+        *path_end = '\0';
+        path_end = (char *)path_tail((char_u *)exe_name);
+        *path_end = '\0';
```

Comment:
I don't understand this: line 568 assigns `path_end`, then line 570 overwrites it without using the result.